content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/maximum-image-age-policy.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Limit the “Build Horizon” of Build Artifacts with the Maximum Image Age Policy"
+type: "article"
+description: "Understanding Identity and Access Management in Chainguard Enforce"
+date: 2023-10-04T15:22:20+01:00
+lastmod: 2023-10-04T15:22:20+01:00
+draft: false
+images: []
+tags: ["policy-controller", "Policies", "Enforce", "Product"]
+menu:
+  docs:
+    parent: "chainguard-enforce-kubernetes"
+weight: 100
+toc: true
+---
+
+The age of your build artifacts can have serious maintenance and security implications for your Kubernetes workloads. Stale software incurs technical debt and is often more likely to contain vulnerabilities that can put your workloads at risk. Additionally, as we discuss in our article [The Principle of Ephemerality](https://www.chainguard.dev/unchained/the-principle-of-ephemerality), refreshing your build artifacts frequently makes it difficult for an attacker to maintain a toehold in your software should they successfully breach your build system without your knowledge.
+
+The importance of freshness applies to a range of software, such as “off-the-shelf” components like monitoring tools, dependencies you compile into your software like base images, and the software you develop and maintain yourself. That said, tracking the age of every piece of software you use in your workloads can be a daunting task. As [even Chainguard’s own security-focused engineers discovered](https://www.chainguard.dev/unchained/conquer-your-build-horizon-with-chainguard-enforce-in-2023), stale software can slip through the cracks despite best efforts.
```

Comment:
Can we soften the last sentence? I think most people may not read the blog and may assume something negative

Reply:
ah, good point! yes, softening the sentence!